sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorRedirectPolicy.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.opentelemetry.exporter;
+
+import com.azure.core.http.HttpPipelineCallContext;
+import com.azure.core.http.HttpPipelineNextPolicy;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.util.logging.ClientLogger;
+import reactor.core.publisher.Mono;
+
+import java.net.HttpURLConnection;
+
+/**
+ * A HttpPipeline policy that retries when a HTTP Redirect is received as response.
+ */
+public final class AzureMonitorRedirectPolicy implements HttpPipelinePolicy {
+
+    private static final int PERMANENT_REDIRECT_STATUS_CODE = 308;
+    private static final int MAX_REDIRECT_RETRIES = 10;
```

Comment:
Why do we want to set this to 10? This is a lot of redirect attempts.

Reply:
If your concern is timeout, I believe timeout is calculated from connection start to a response. If the SDK is handling looking at the response code and then attempting another connection (which I think is true), a new timeout starts.  So 10 redirects should be OK.  Do you suggest we keep this configurable with a default of 10?